pkg/api/types.go

```diff
@@ -1202,6 +1208,21 @@ type KeyToPath struct {
 	Mode *int32
 }
 
+// LocalVolume represents a local volume accessible only from one node
+type LocalVolumeSource struct {
+	// NodeName represents the node that the volume is located at
+	NodeName string
+	// Fs represents a filesystem-based local volume
+	// +optional
+	Fs *LocalFsVolume
```

Comment:
> but we're not sure yet if there may be any other additional fields required

That's ok. Path will be forwards compatible right?

> We are looking at exposing PVs as block devices in the future

If the assumption is that PV is representing an attribute of the underlying storage type, then that's OK. In theory, a storage type should specify it's access type. A PV is merely a wrapper around an existing storage type.